dhall/src/Dhall/Import.hs

```diff
@@ -786,16 +800,20 @@ fetchFresh (Env env) = do
 
 fetchFresh Missing = throwM (MissingImports [])
 
+fetchDisabledForHeaders :: URL -> StateT Status IO Data.Text.Text
```

Comment:
Correct me if I'm wrong, but I believe you no longer need to disable fetch for headers since we can now lean on cyclic import detection

However, that would entail a matching change to how we fetch the headers, which is to have the `loadSiteHeaders` subroutine reuse the ambient `Status` (see my comment below)